app/src/main/res/values/strings.xml

```diff
@@ -91,6 +91,7 @@
   <string name="chapter_completed">Chapter %s with title %s is completed</string>
   <string name="chapter_in_progress">Chapter %s with title %s is in progress</string>
   <string name="chapter_prerequisite_title_label">Complete Chapter %s: %s to unlock this chapter.</string>
+  <string name="chapter_prerequisite_title_label_without_chapter_title">Complete the previous chapter to unlock this chapter</string>
```

Comment:
```suggestion
  <string name="chapter_prerequisite_title_label_without_chapter_title">Complete the previous chapter to unlock this chapter.</string>
```

For consistency with the other label.

Reply:
I have made this change. It should reflect on the code now.